src/components/TaskCard/AddTaskCard.jsx

```diff
@@ -1,17 +1,22 @@
 import React, { useState } from "react";
 import { useDispatch } from "react-redux";
 import PropTypes from "prop-types";
-import { sanitizeText } from "utils/helpers/sanitizeText.js";
-import { addTask } from "store/actions/";
+import { addTask, searchTask } from "store/actions/";
 import "components/TaskCard/index.scss";
 import {
   ICON_DELETE,
   DELETE_ICON_ALT_TEXT,
   KEY_ENTER,
   FILTER_STATE_ALL,
 } from "utils/constant";
+import { sanitizeText } from "utils/helpers/sanitizeText";
 
-const AddTaskCard = ({ isCardCreated, onCreateCard, setFilter }) => {
+const AddTaskCard = ({
+  onSearchBarVisible,
+  isCardCreated,
+  onCreateCard,
+  onFilterState,
```

Comment:
Can we re-order the props as below
1. data props
2. events

```suggestion
  isCardCreated,
  onSearchBarVisible,
  onCreateCard,
  onFilterState,
```